core/domain/protofiles/text_classifier.proto

```diff
@@ -0,0 +1,21 @@
+// coding: utf-8
+//
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+message TextClassifier {
```

Comment:
I'm not keen on this name. A classifier to me seems like an actual object with functionality (e.g. to classify something). My understanding is that this is actually the holder for a text classifier's model. Perhaps ``TextClassifierModel`` would be clearer?

Reply:
Good catch. Howe about `TextClassifierFrozenModel` as we use frozen model term to specify the parameters exported by the model which are necessary for inference in frontend.

Reply:
Done.

Reply:
Frozen model sounds good--thanks!